plugins/signalmonitor/signalhistorymodel.h

```diff
@@ -64,6 +65,7 @@ class SignalHistoryModel : public QAbstractTableModel
 
 public:
     enum ColumnId {
+        MonitoredColumn = 0,
```

Comment:
Its width is roughly "X M. ^", so yeah something like 3 times of the width of the check indicator itself, not so much in my opinion, still it depend which screen size you use :)
This column add some benefit to the view, ie, you can sort by checked rows and still be able to sort by name.
That can be useful to uncheck rows that are not contiguous, it can become very boring to scroll all the view to find the missing checked row (ok, you can click the header check indicator to check all then unselect all... at the cost of some network traffic).
If it really broke your daily confort usage within this inspector, then of course we can merge it.